src/main/java/core/basesyntax/files/input_file.csv

```diff
@@ -0,0 +1,9 @@
+type,fruit,quantity
```

Comment:
According the common mistakes please place the input and output files into the `src/main/resources folder`.